### ISSUE TITLE
1.3.2 seems to have a breaking change: no method named `map_or` found for type `std::result::Result<std::fs::FileType, std::io::Error>`

### ISSUE THREAD

**Issue by crusty-dave** [nabijaczleweli/rust-embed-resource#18]
1.3.2 seems to have a breaking change.  Note that I am building with rustc 1.40.0.

```
  error[E0599]: no method named `map_or` found for type `std::result::Result<std::fs::FileType, std::io::Error>` in the current scope
     --> c:\staging\cargo\...\.cargo\registry\src\ssd-git.juniper.net-4dc6a241367bbfe0\embed-resource-1.3.2\src\windows_msvc.rs:165:70
      |
  165 |     read_dir.filter_map(|dir| dir.ok()).filter(|dir| dir.file_type().map_or(false, |ft| ft.is_dir()))
      |                                                                      ^^^^^^ help: there is a method with a similar name: `map_err`
  
  For more information about this error, try `rustc --explain E0599`.
```

**Comment by nabijaczleweli** [comment]
I made no MSRV-related guarantees before but you're right, I'll drop it to 1.35.0, like `cargo-update`.

**Comment by nabijaczleweli** [comment]
Thanks for the report!

Released in [`v1.3.3`](https://github.com/nabijaczleweli/rust-embed-resource/releases/tag/v1.3.3).
